File: solutions/0268_missing-number.java
// Problem: https://leetcode.com/problems/missing-number
// Runtime: 0 ms

class Solution {
    public int missingNumber(int[] nums) {
        int totTrue = 0;
        int tot = 0;
        int n = nums.length;
        for (int i = 0; i < n; i++) {
            totTrue += i;
            tot += nums[i];
        }
        return n + totTrue - tot;
    }
}

File: solutions/0268_missing-number.py
# Problem: https://leetcode.com/problems/missing-number

class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        totalTrue = 0
        total = 0
        n = len(nums)
        for i in range(n):
            totalTrue += i
            total += nums[i]
        return n + totalTrue - total